test/test-integration-3rd/src/test/java/com/alipay/sofa/rpc/client/ElasticConnectConnectionHolderTest.java

```diff
@@ -0,0 +1,130 @@
+
+    private static ServerConfig   serverConfig1;
+    private static ServerConfig   serverConfig2;
+    private static ServerConfig   serverConfig3;
+    private static ServerConfig   serverConfig4;
+    private static RegistryConfig registryConfig;
+
+    @BeforeClass
+    public static void startServer() throws Exception {
+
+        registryConfig = new RegistryConfig()
+            .setProtocol("zookeeper")
+            .setAddress("127.0.0.1:2181");
+
+        serverConfig1 = new ServerConfig()
+            .setStopTimeout(0)
+            .setPort(22223);
+        ProviderConfig<HelloService> providerConfig = new ProviderConfig<HelloService>()
+            .setInterfaceId(HelloService.class.getName())
+            .setRef(new HelloServiceImpl())
+            .setServer(serverConfig1)
+            .setRepeatedExportLimit(-1)
+            .setRegistry(registryConfig);
+        providerConfig.export();
+
+        serverConfig2 = new ServerConfig()
+            .setStopTimeout(0)
+            .setPort(22224);
+        ProviderConfig<HelloService> providerConfig2 = new ProviderConfig<HelloService>()
+            .setInterfaceId(HelloService.class.getName())
+            .setRef(new HelloServiceImpl())
+            .setServer(serverConfig2)
+            .setRepeatedExportLimit(-1)
+            .setRegistry(registryConfig);
+        providerConfig2.export();
+
+        serverConfig3 = new ServerConfig()
+            .setStopTimeout(0)
+            .setPort(22225);
+        ProviderConfig<HelloService> providerConfig3 = new ProviderConfig<HelloService>()
+            .setInterfaceId(HelloService.class.getName())
+            .setRef(new HelloServiceImpl())
+            .setServer(serverConfig3)
+            .setRepeatedExportLimit(-1)
+            .setRegistry(registryConfig);
+        providerConfig3.export();
+
+        serverConfig4 = new ServerConfig()
+            .setStopTimeout(0)
+            .setPort(22226);
+        ProviderConfig<HelloService> providerConfig4 = new ProviderConfig<HelloService>()
+            .setInterfaceId(HelloService.class.getName())
+            .setRef(new HelloServiceImpl())
+            .setServer(serverConfig4)
+            .setRepeatedExportLimit(-1)
+            .setRegistry(registryConfig);
+        providerConfig4.export();
+    }
+
+    @AfterClass
+    public static void stopServer() {
+        serverConfig1.destroy();
+        serverConfig2.destroy();
+        serverConfig3.destroy();
+        serverConfig4.destroy();
+    }
+
+    @Test
+    public void testConnectElatic() throws Exception {
+
+        //please change the connect.elatic.size or connect.elatic.precent in the rpc-config.json
+        //use connect.elatic.precent first unless connect.elatic.precent = 0
+
+        ConsumerConfig<HelloService> consumerConfig = new ConsumerConfig<HelloService>()
+            .setInterfaceId(HelloService.class.getName())
+            .setConnectionHolder("elatic")
```

Comment:
Run test case failure because this typo "elatic" .

Reply:
thank you for your help